values.yaml

```diff
@@ -128,6 +136,14 @@ global:
           _default: hostpath
           local: hostpath
           aws: efs-sc
+        auto:
+          _default: false
+          local: false
+          aws: false
+        mode:
+          _default: ReadWriteOnce
+          local: ReadWriteMany
+          aws: ReadWriteMany
```

Comment:
Ditto here, should only need to be ReadOnlyMany

Reply:
as above.